python/cudf/cudf/_lib/cpp/io/csv.pxd

```diff
@@ -125,115 +125,115 @@ cdef extern from "cudf/io/csv.hpp" \
+        string get_na_rep() except +
+        bool is_enabled_include_header() except +
+        size_type get_rows_per_chunk() except +
+        string get_line_terminator() except +
+        char get_inter_column_delimiter() except +
+        string get_true_value() except +
+        string get_false_value() except +
 
         # setter
-        void set_metadata(cudf_io_types.table_metadata* val) except+
-        void set_na_rep(string val) except+
-        void enable_include_header(bool val) except+
-        void set_rows_per_chunk(size_type val) except+
-        void set_line_terminator(string term) except+
-        void set_inter_column_delimiter(char delim) except+
-        void set__true_value(string val) except+
-        void set_false_value(string val) except+
+        void set_metadata(cudf_io_types.table_metadata* val) except +
+        void set_na_rep(string val) except +
+        void enable_include_header(bool val) except +
+        void set_rows_per_chunk(size_type val) except +
+        void set_line_terminator(string term) except +
+        void set_inter_column_delimiter(char delim) except +
+        void set__true_value(string val) except +
+        void set_false_value(string val) except +
 
         @staticmethod
         csv_writer_options_builder builder(
             cudf_io_types.sink_info sink,
             cudf_table_view.table_view table
-        ) except+
+        ) except +
 
     cdef cppclass csv_writer_options_builder:
-        csv_writer_options_builder() except+
+        csv_writer_options_builder() except +
         csv_writer_options_builder(
             cudf_io_types.sink_info sink,
             cudf_table_view.table_view table
-        ) except+
+        ) except +
 
         csv_writer_options_builder& metadata(
             cudf_io_types.table_metadata* val
-        ) except+
-        csv_writer_options_builder& na_rep(string val) except+
-        csv_writer_options_builder& include_header(bool val) except+
-        csv_writer_options_builder& rows_per_chunk(size_type val) except+
-        csv_writer_options_builder& line_terminator(string term) except+
-        csv_writer_options_builder& inter_column_delimiter(char delim) except+
-        csv_writer_options_builder& true_value(string val) except+
-        csv_writer_options_builder& false_value(string val) except+
-
-        csv_writer_options build() except+
+        ) except +
+        csv_writer_options_builder& na_rep(string val) except +
+        csv_writer_options_builder& include_header(bool val) except +
+        csv_writer_options_builder& rows_per_chunk(size_type val) except +
+        csv_writer_options_builder& line_terminator(string term) except +
+        csv_writer_options_builder& inter_column_delimiter(char delim) except +
+        csv_writer_options_builder& true_value(string val) except +
+        csv_writer_options_builder& false_value(string val) except +
+
+        csv_writer_options build() except +
```

Comment:
Yes, sort of. It might have been for E275 — but we can’t solve all of those because of the syntax for `except?` (`except ?` also works for Cython but triggers some other error iirc) so I fixed the ones that we could fix and disabled that rule for Cython. At the very least, this establishes consistent spacing for all our instances of `except +` instead of `except+`.